docs/configuration.md

```diff
@@ -760,3 +760,12 @@ These timezones will appear in the dropdown instead of the default, which are
 
 `['America/Juneau', 'America/Los_Angeles', 'America/Yellowknife', 'America/Phoenix', 'America/Denver', 'America/Mexico_City', 'America/Chicago', 'America/New_York', 'America/Argentina/Buenos_Aires', 'Etc/UTC',
 'Asia/Jerusalem', 'Europe/Paris', 'Asia/Kathmandu', 'Asia/Hong_Kong', 'Asia/Seoul', 'Pacific/Guam']`
+
+### Sentry DSN
+
+Add you Sentry DSN to report errors to Sentry
```

Comment:
Typo: "your" or just "Sentry DSN ...". Link to the Sentry documentation https://docs.sentry.io/clients/java/config/